android/library/src/main/java/io/logto/android/authflow/webview/WebViewAuthFlow.kt

```diff
@@ -75,28 +76,31 @@ class WebViewAuthFlow {
         authorizationCode: String,
         authenticationCallback: AuthenticationCallback
     ) {
-        val call = logtoClient.getCredential(
+        MainScope().launch {
+            try {
+                authenticationCallback.onSuccess(fetchCredential(authorizationCode))
```

Comment:
Looks a little bit weird. But no blocking. Should be good after refactoring the callback approach.

Reply:
I will learn more about it